patches/components-sync-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
+index 0606b7e68670877100114732174af25ad0b42f77..22a62cebbd47644c4f189be012134f7c04c92af4 100644
+--- a/components/sync/BUILD.gn
++++ b/components/sync/BUILD.gn
+@@ -99,6 +99,8 @@ jumbo_static_library("base") {
+   ]
+   deps = [
+     "//base:i18n",
++    "//brave/components/brave_sync:jslib_messages",
```

Comment:
stuff like this should always be a single dep that encapsulates everything we need. There is never a reason why we should patch more than one dep into the same target